Storage/SQL/SQLiteRemoteClientsAndTabs.swift

```diff
@@ -13,11 +13,10 @@ open class SQLiteRemoteClientsAndTabs: RemoteClientsAndTabs {
     let db: BrowserDB
     let clients = RemoteClientsTable<RemoteClient>()
     let tabs = RemoteTabsTable<RemoteTab>()
-    let commands = SyncCommandsTable<SyncCommand>()
 
     public init(db: BrowserDB) {
         self.db = db
-        _ = self.db.createOrUpdate(clients, tabs, commands)
+        _ = self.db.createOrUpdate(clients, tabs)
```

Comment:
I don't see how this works. There's _one_ version number in the store, and it's in use to track whether `BrowserTable` has inited…